docs/reference/objects/accommodation/availabilty_day.md

```diff
@@ -8,7 +8,7 @@ When using an `availability_day` object you have access to the following attribu
 
 # availability_day.prices
 
-Returns an array of `variant_prices` for a day.
+Returns an array of [variant_prices]({% docs/reference/objects/product/variant_price.md %}) for a day.
```

Comment:
This is throwing an error for me. You need to include the `link` tag, e.g.

```
[variant_prices]({% link docs/reference/objects/product/variant_price.md %})
```

Also, I think you need to rebase with the latest, as it's complaining that `variant_price.md`.

```
Could not find document 'docs/reference/objects/product/variant_price.md' in tag 'link'. (ArgumentError)
```

You can use `bundle exec jekyll serve` to run it locally to check it all works properly.

Reply:
Ah I think I messed this up in my last rebase, good spot!

Reply:
Hmm the variant prices are still missing so might need to try a new branch and cherry pick

Reply:
🤦 we don't have the variant price drop in so I don't know what I was linking to previously...